documentation/how-to/tracking.en.md

```diff
@@ -5,89 +5,91 @@ tx_slug: documentation_how-to_tracking
-button to begin recording. At this stage, a feature form will appear, which allows
-you to define the attributes of the feature(s) that will be created while QField
-tracks your position.
+    !![](../assets/images/tracking-settings.png,350px)
 
-For line and polygon layers, a single feature - its geometry formed of vertices
-from the recorded positions - will be create per tracking session.  For point
-layers, a feature will be created for each recorded position, with attribute
-values entered in the form remembered across features.
+    In the case that you wish to ensure that the next vertex is set not too far away from the most recent one (for example if you are in steep terrain and elevation is relevant), you can set a maximum tolerance distance.
 
-To prevent overlapping, you have the option to set a maximum distance tolerance between the last and next tracked point/vertices. This ensures that if the position exceeds this defined distance, it is considered as an erroneous entry.
+    !![](../assets/images/maximum-distance-tolerance.png,350px)
 
-!!! note
-    QField will skip the feature form step if the layer is configured to hide all attributes or if form has been set to hide on feature addition in the attribute form settings. This can streamline the process of starting tracking sessions.
+3. Tap the **Start tracking** button to begin the tracking.
+4. Set the attributes for the feature(s) about to be created.
 
 !!! note
-    QField offers the capability to resume tracking sessions seamlessly, even if the application has been restarted or the device has been rebooted. This ensures continuity by allowing previously started tracking sessions to be reactivated without loss progress.
+    For line and polygon layers, a single feature will be created per tracking session, with its geometry formed from the recorded positions.
+    For point layers, a new feature will be created for each recorded position, and the attribute values entered in the form will be remembered for subsequent points.
+
+!!! tip
+    QField will skip the feature form if the layer is configured to hide all attributes or if the form has been set to hide on feature addition in the attribute form settings.
+    This can streamline the process of starting tracking sessions.
+
+## Resuming to a Previous Session
 
-!![](../assets/images/maximum-distance-tolerance.png,550px)
+If did not explicitly terminate your tracking session (for example, you closed the app), QField allows you to continue where you left off.
+A prompt will appear asking whether to **resume the last session** or **start a new one**.
+
+- **Resuming** a session for a line or polygon layer will continue adding vertices to the feature from the previous session.
+This allows you to simply continue across app restarts.
+
+- **Starting a new session** will discard the incomplete feature from the previous session and begin a new one.
```

Comment:
There again we jump from paragraphs vs bullet point (in this case to say the same thing as what we said in the paragraph above :) )